### DISCUSSION TITLE
anasoid/jmeter:latest keep crashing in Kubernetes

### DISCUSSION THREAD

**Discussion by AntonioSun** [anasoid/docker-jmeter#64] [Q&A] [answered]
`kubectl get pod docker-jmeter -o jsonpath='{.status.containerStatuses}' | jq` gets me:

```json
[
  {
    "containerID": "containerd://8ed71c7a0a48f3b552f998135258751104e6819cf5b2163a7d6036ced67d619e",
    "image": "docker.io/anasoid/jmeter:latest",
    "imageID": "docker.io/anasoid/jmeter@sha256:cbe99c7ea15ebf890fb2bf2556e62e334b8d832949a59afbdf8ccb6698b17b4f",
    "lastState": {
      "terminated": {
        "containerID": "containerd://8ed71c7a0a48f3b552f998135258751104e6819cf5b2163a7d6036ced67d619e",
        "exitCode": 0,
        "finishedAt": "2022-01-25T20:09:59Z",
        "reason": "Completed",
        "startedAt": "2022-01-25T20:09:41Z"
      }
    },
    "name": "docker-jmeter",
    "ready": false,
    "restartCount": 5,
    "started": false,
    "state": {
      "waiting": {
        "message": "back-off 2m40s restarting failed container=docker-jmeter pod=docker-jmeter_jmeter(7ca4da24-622e-47fd-83b3-d125ea6f037e)",
        "reason": "CrashLoopBackOff"
      }
    }
  }
]
```

Tried many times and the results are always the same.

**Comment by anasoid** [comment, marked as answer]
have you mount  /jmeter/project ??

can you give me the log of pod ??

**Reply by AntonioSun** [reply]
Bingo! I didn't mount /jmeter/project.

If it happens again, how to get the log of pod? thx

**Reply by anasoid** [reply]
to have  logs

for the log : kubectl logs -n <namespace> <podname>


I'm working on new feature (will be released today or tomorrow max), related to Kubernetes, it will be possible to have a ready check on file before starting, like this you can : 
1. Start jmeter container with a ready wait on jmx file or other file
2. copy file to container after starting it.

**Reply by anasoid** [reply]
new change is available to check ready see example on https://github.com/anasoid/docker-jmeter#using-wait-to-be-ready
